COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -6,6 +6,7 @@ After=syslog.target
 ExecStart=/bin/sh -c "/bin/evmserver.sh start"
 ExecStop=/bin/sh -c "/bin/evmserver.sh stop"
 Type=forking
+Restart=on-failure
```

Comment:
Looks like there is no more recent version of systemd out there for Centos7.  I attempted to install from source and the appliance would only boot into emergency mode after the install due to some device timeouts that I don't see with the older version. So it looks like we are stuck here until the Centos rpm gets updated.